src/services/routes/org.routes.spec.ts

```diff
@@ -16,20 +28,29 @@ describe('org routes', () => {
       url: '/orgs',
     };
 
-    test('Valid parameters with non ASCII name should return success', async () => {
+    test('Valid parameters should return urls', async () => {
```

Comment:
```suggestion
    test('Valid parameters should return URLs', async () => {
```

Just for consistency with other repos.